test/optim/test_optim.py

```diff
@@ -770,11 +770,10 @@ def _test_derived_optimizers(self, optimizer_pairs_with_flags, flag, reduced_pre
                 # foreach/fused optimizers should be tested with a param_groups['params'] with
                 # zero_size tensor as its last param.
                 # ref: https://github.com/pytorch/pytorch/issues/100701
-                empty_params = [torch.empty((), device=device, dtype=torch.float64)]
+                empty_param = torch.empty((), device=device, dtype=torch.float64)
+                parameters = list(model.parameters()) + [empty_param]
 
-                optimizer = optimizer_constructor(
-                    list(model.parameters()) + empty_params, **params_with_flags
-                )
+                optimizer = optimizer_constructor(parameters, **params_with_flags)
 
```

Comment:
This is just refactoring right?

Reply:
This allows us to access the step of the first param later :p